### PR TITLE
WIP: Scala 2.13

### LINKED ISSUE #137: 2.13 support
Hi! Is there some roadmap for 2.13 support?

### PR DESCRIPTION
**Pull request by fdietze**
fixes #137 

(currently blocked by https://github.com/scala/scala-collection-compat/issues/247)

### CODE REVIEW

**Comment by mathieuleclaire** [comment]
All the tests are successfull in sbt. I only see Symbol deprecation. What is still in progress ?

**Comment by fdietze** [comment]
The cross-compilation with 2.12 and 2.11 doesn't work yet. See:
https://github.com/scala/scala-collection-compat/issues/247
https://github.com/scala/scala-collection-compat/issues/247

**Comment by mathieuleclaire** [comment]
Yes, it's what i realized looking to the CI logs. So we just need to wait for an update of the collection-compat lib if I understand properly ?

**Comment by fdietze** [comment]
Correct. I hope that this PR fixes it: https://github.com/scala/scala-collection-compat/pull/249
I'll leave another message there.

**Comment by mathieuleclaire** [comment]
collection-compat has been released !
https://github.com/scala/scala-collection-compat/releases

**Comment by fdietze** [comment]
Looks like we have to drop Scala 2.11 now. Since libraries like `acyclic` and `monocle` dropped 2.11 in favor of 2.13:
https://mvnrepository.com/artifact/com.lihaoyi/acyclic
https://mvnrepository.com/artifact/com.github.julien-truffaut/monocle-macro

**Comment by fdietze** [comment]
@Voltir @lihaoyi How do you feel about dropping 2.11 support in favor of 2.13?

**Comment by Voltir** [comment]
@fdietze I am for it

**Comment by mathieuleclaire** [comment]
@fdietze let's drop 2.11 !

File: build.sbt [hunk 1]
```diff
@@ -17,14 +17,17 @@ val sharedSettings = Seq(
...
     "-Xfuture" ::
-    "-Ypartial-unification" ::
-    "-Yno-adapted-args" ::
-    "-Ywarn-infer-any" ::
-    "-Ywarn-value-discard" ::
-    "-Ywarn-nullary-override" ::
-    "-Ywarn-nullary-unit" ::
     (CrossVersion.partialVersion(scalaVersion.value) match {
-      case Some((2, v)) if v >= 12 =>
+      case Some((2, 12)) =>
```
Comment (fdietze) [inline build.sbt]: I'd like to have a review on this one.

What is a good approach to handle compiler options when cross-compiling?

**Pull request merged**